easylist/easylist_general_hide.txt

```diff
@@ -9234,6 +9234,7 @@
 ##.SponsoredLinksPanel
 ##.SponsoredResults
 ##.Sponsored_link
+##.sponsors
```

Comment:
Might be too generic, you could make this specific by adding it here :

https://github.com/easylist/easylist/blob/038e8259f4bbc0f4ab94ffb30b8125c67de7ce2b/easylist/easylist_specific_hide.txt#L1996

Reply:
I was thinking about whether it would be to generic...

I'll move it a bit later :smile: 

Suger fingers...